tests/core/full_node/test_mempool.py

```diff
@@ -2183,6 +2183,7 @@ class TestMaliciousGenerators:
             ConditionOpcode.ASSERT_SECONDS_RELATIVE,
         ],
     )
+    @pytest.mark.benchmark
```

Comment:
I see. You had the funny choice of whether to mark the benchmarks, or mark the "not benchmarks"

Reply:
I think it's important that tests not be considered benchmarks by default